### PR TITLE
Update the latest experiments run on SL hardware.

### PR DESCRIPTION
**Pull request by kusumachalasani**

### CODE REVIEW

**Comment by dinogun** [comment]
@kusumachalasani As a general rule, please post one experiment per PR as soon as the experiment is completed. This is difficult to review as well as digest the results

File: techempower/README.md [hunk 1]
```diff
@@ -1,13 +1,58 @@
...
+[Hotspot Layer]   AlwaysPreTouch [false]			- Force all freshly committed pages to be pre-touched.
+[Hotspot Layer]   AlwaysTenure [false]       			- Always tenure objects in eden (ParallelGC only).
+[Hotspot Layer]   BackgroundCompilation [true]  		- A thread requesting compilation is not blocked during compilation.
+[Hotspot Layer]   DoEscapeAnalysis [true]   			- Perform escape analysis.
+[Hotspot Layer]   UseInlineCaches [true]   			- Use Inline Caches for virtual calls.
+[Hotspot Layer]   UseLoopPredicate [true]   			- Generate a predicate to select fast/slow loop versions.
+[Hotspot Layer]   UseStringDeduplication [false]  		- Use string deduplication.
+[Hotspot Layer]   UseSuperWord [true]    			- Transform scalar operations into superword operations.
+[Hotspot Layer]   UseTypeSpeculation [true]    			- Speculatively propagate types from profiles.
+
```
Comment (dinogun) [inline techempower/README.md:34]: Good summary!

File: techempower/experiment-10/best-result-config-under-100trials.yaml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+                        limits:
+                          cpu: 4
+                          memory: 2374Mi
+            - name: update env
+              spec:
+                template:
+                  spec:
+                    container:
+                      env:
+                        JAVA_OPTIONS: i>-
```
Comment (dinogun) [inline techempower/experiment-10/best-result-config-under-100trials.yaml:61]: Is this international => `i>-`

File: techempower/experiment-11/best-result-config.yaml [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+              datasource: prometheus
+          config:
+            - name: update requests and limits
+              spec:
+                template:
+                  spec:
+                    container:
+                      resources:
+                        requests:
+                    i      cpu: 3.75
```
Comment (dinogun) [inline techempower/experiment-11/best-result-config.yaml:50]: stray character

File: techempower/experiment-12/best-result-config.yaml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+              datasource: prometheus
+          config:
+            - name: update requests and limits
+              spec:
+                template:
+                  spec:
+                    container:
+                      resources:
+                        requests:
+                    i      cpu: 4
```
Comment (dinogun) [inline techempower/experiment-12/best-result-config.yaml:50]: stray character

Comment (kusumachalasani) [reply techempower/experiment-12/best-result-config.yaml]: corrected it.

File: techempower/experiment-10/best-result-config-under-100trials.yaml (hunk 2 above)
Comment (kusumachalasani) [reply techempower/experiment-10/best-result-config-under-100trials.yaml:61]: removed character i which is a typo.

File: techempower/experiment-11/best-result-config.yaml (hunk 3 above)
Comment (kusumachalasani) [reply techempower/experiment-11/best-result-config.yaml]: corrected it.

**Review by dinogun — COMMENTED** [review]
@kusumachalasani The percentage graph with logarithmic scale is looking much better. Couple of small changes, It should be <1ms instead of 1ms, also add data points

**Review by dinogun — APPROVED** [review]
LGTM

**Pull request merged**